curated_transformers/tokenization/tokenizer.py

```diff
@@ -205,3 +217,51 @@ def __call__(self, chunks: Iterable[InputChunks]) -> List[InputChunks]:
                 )
             )
         return bos_eos_chunks
+
+
+class DefaultNormalizer(Normalizer):
+    """
+    Performs normalization operations on input chunks before encoding.
+    """
+
+    def __init__(
+        self,
+        *,
+        utf_normalization: Optional[str] = None,
```

Comment:
Since there are only four forms, maybe we want an enum? If not, we should maybe enumerate the options in the docs.